### DISCUSSION TITLE
blog/2023/05/how-to-build-a-fully-internationalized-nextjs-application-using-lingui-v4

### DISCUSSION THREAD

**Discussion by giscus** [bravo-kernel/bravo-kernel.com#176] [Comments]
# blog/2023/05/how-to-build-a-fully-internationalized-nextjs-application-using-lingui-v4

https://bravo-kernel.com/blog/2023/05/how-to-build-a-fully-internationalized-nextjs-application-using-lingui-v4

**Comment by keiraarts** [comment]
Thank you!
